File: src/BisonMeat.cpp
#include "../include/BisonMeat.hpp"


//Constructor menerima harga, dan mempassing producer untuk superclass.
BisonMeat::BisonMeat(int _price, string _producer){
	initializeFarmProduct(_producer);
	initializeProduct(_price, FARMPRODUCT, BISON_MEAT, "Daging Bison");
}

File: Java/src/main/java/engisubs/gameobject/product/farm_product/meat/BisonMeat.java
package main.java.engisubs.gameobject.product.farm_product.meat;

public class BisonMeat extends Meat{
    public BisonMeat(int _price, String _producer){
        super.initializeFarmProduct(_producer);
        super.initializeProduct(_price, ProductType.FARMPRODUCT, GameObjectType.BISON_MEAT, "Daging Segar BISON");
    }
}